File: 백준/Bronze/11720. 숫자의 합/숫자의 합.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] ars) throws IOException {
        BufferedReader bf = new BufferedReader(new InputStreamReader(System.in));

        int n = Integer.parseInt(bf.readLine());
        int sum = 0;
        String st= bf.readLine();
        char[] array = st.toCharArray();

        for (int j = 0; j < n; j++) {
            sum += array[j] - 48;
        }

        System.out.println(sum);
    }

}

File: 백준/Bronze/11720. 숫자의 합/숫자의 합.cc
#include <stdio.h>
#include <string.h>
int main() {

	int n1, sum = 0;
	char n2[100] = "";

	scanf("%d", &n1);
	scanf("%s", n2);

	for (int i = 0; i < n1; i++) {
		sum += n2[i] - 48;
	}

	printf("%d", sum);



}